### DISCUSSION TITLE
How to remove keyboard after user's response

### DISCUSSION THREAD

**Discussion by SwapnilSoni1999** [telegraf/telegraf#1453] [General]
I've passed 

```js
await ctx.reply(':)', { reply_markup: Markup.removeKeyboard() })
```

but it is still showing the keyboard 
![image](https://user-images.githubusercontent.com/25403317/117656611-310c8380-b1b6-11eb-9c15-3375a2b056a2.png)

**Comment by dcdunkan** [comment]
It should be just `Markup.removeKeyboard()` without `reply_markup`. Works for me! Hope it helps.
```javascript
await ctx.reply(":)", Markup.removeKeyboard());
